plugins/vpp/ifplugin/data_resync.go

```diff
@@ -600,8 +600,8 @@ func (plugin *NatConfigurator) resolveMappings(nbDNatConfig *nat.Nat44DNat_DNatC
 				for _, nbLocal := range nbMapping.LocalIps {
 					var found bool
 					for _, vppLocal := range vppLbMapping.LocalIps {
-						if nbLocal.LocalIp == vppLocal.LocalIp || nbLocal.LocalPort == vppLocal.LocalPort ||
```

Comment:
This seems like bug.. right?

Reply:
yep, bug. Please fix in pantheon-dev too